apps/condo/domains/user/components/containers/PosterLayout.tsx

```diff
@@ -14,12 +15,21 @@ interface IPosterLayoutProps {
 }
 const TYPOGRAPHY_CONTACT_STYLE: React.CSSProperties = { color: colors.textSecondary }
 const ROW_STYLE = { margin: '65px 0 65px', justifyContent: 'center' }
+const RE_CAPTCHA_PARAGRAPH = { color: colors.textSecondary, width: '100%' }
```

Comment:
Paragraph must be already using width: 100%....and you can specify on `type: secondary` on it